website/app/controllers/application.js

```diff
@@ -1,15 +1,44 @@
 import Controller from '@ember/controller';
 import { action } from '@ember/object';
 import { tracked } from '@glimmer/tracking';
+import { scheduleOnce } from '@ember/runloop';
+import { inject as service } from '@ember/service';
 
 export default class ApplicationController extends Controller {
-  @tracked showSidebarOnSmallViewport = false;
+  @service router;
+  @service fastboot;
 
-  // TODO add a reset for the `showSidebarOnSmallViewport` on route change
+  // notice: don't set it to false, otherwise the sidebar menu will appear open on page load
+  @tracked isSidebarVisibleOnSmallViewport;
+
+  constructor() {
+    super(...arguments);
+    this.router.on('routeDidChange', this, 'routeDidChange');
+  }
+
+  routeDidChange() {
```

Comment:
I think this is also getting triggered on query param "transition", hence toggling the sidebar back and forth per my comment on the main thread